WordPress/src/main/res/layout/history_details_activity.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
```suggestion
<?xml version="1.0" encoding="utf-8"?>

<FrameLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:id="@+id/fragment_container"
    android:layout_height="match_parent"
    android:layout_width="match_parent" >
</FrameLayout>
```

Since we only need the `FrameLayout` view to insert the `HistoryDetailsContainerFragment`, the `LinearLayout` view can be removed completely.